ui/.env.goerli

```diff
@@ -1,5 +1,2 @@
 NEXT_PUBLIC_CHAIN=goerli
-NEXT_PUBLIC_AGREEMENT_STATEMENT=By claiming a Nation3 passport I agree to the terms defined in the following URL
-NEXT_PUBLIC_AGREEMENT_URL=https://bafkreiadlf3apu3u7blxw7t2yxi7oyumeuzhoasq7gqmcbaaycq342xq74.ipfs.dweb.link
-NEXT_PUBLIC_VENATION_REQUIRED_STAKE=2
-NEXT_PUBLIC_VENATION_REWARDS_MULTIPLIER=2.5
\ No newline at end of file
+NFTSTORAGE_KEY=
```

Comment:
@xPi2 Is there a way to test the NFT storage functionality on Goerli?

Reply:
I think it is the same as mainnet, create an account here https://nft.storage/  and add an API key, and once we sign a message on the join page, check if that signature is stored in https://nft.storage/files/

Reply:
@biendeveloper Okay, I created a test API key and added it to `.eng.goerli`

Reply:
Yep correct :)